cpp/include/phevaluator/card.h

```diff
@@ -23,6 +23,23 @@
 #include <string>
 
 namespace phevaluator {
+const static std::unordered_map<char, int> rankMap = {
+  {'2', 0}, {'3', 1}, {'4', 2}, {'5', 3},
+  {'6', 4}, {'7', 5}, {'8', 6}, {'9', 7},
+  {'T', 8}, {'J', 9}, {'Q', 10}, {'K', 11}, {'A', 12},
+};
+const static std::unordered_map<char, int> suitMap = {
+  {'C', 0}, {'D', 1}, {'H', 2}, {'S', 3},
+  {'c', 0}, {'d', 1}, {'h', 2}, {'s', 3},
+};
+const static std::unordered_map<int, char> rankReverseMap = {
```

Comment:
Since the indexes are ranged from 0 to 12, do you think we can replace the `unordered_map` with `array`?

Reply:
Good idea. I modified it.